styles/linter-plus.less

```diff
@@ -15,6 +15,9 @@ linter-message {
   overflow-y: scroll;
   max-height: 150px;
   padding: 4px 8px;
+  &[hidden]{
```

Comment:
`<div id="test" hidden></div>`
```css
#test{ display: block; }
```
Now that test div won't be hidden anymore despite having that hidden attribute, so we have to add override it again
```css
#test{ display: block; }
#test[hidden] { display: none  }
```